ci/ray_ci/automation/filter_flaky_tests.py

```diff
@@ -1,29 +1,30 @@
 import sys
+import click
 
-from ci.ray_ci.utils import filter_out_flaky_tests
+from ci.ray_ci.utils import filter_flaky_tests
 from ray_release.configs.global_config import init_global_config
 from ray_release.bazel import bazel_runfile
 
 
-def main():
+@click.command()
+@click.argument("prefix", required=True)
+@click.argument(
+    "filter_option", required=True, type=click.Choice(["only-flaky", "non-flaky"])
```

Comment:
how about we change `filter_option` to `state_filter`? and the two options to `flaky` and `-flaky`?